api/src/main/java/jakarta/data/repository/Repository.java

```diff
@@ -26,13 +26,36 @@
 /**
  * <p>Annotates a data repository interface that will be implemented by the container/runtime.</p>
  *
+ * <p>The application defines entity classes to represent data in the database
+ * and defines repository interface methods that perform queries and other
+ * operations on entities. Repository interfaces are annotated with this annotation.
+ * Entities are simple Java objects with fields or accessor methods
+ * designating each entity property.</p>
+ *
+ * <p>Jakarta Persistence and Jakarta NoSQL define entity models that you
+ * can use in Jakarta Data. You may use {@code jakarta.persistence.Entity}
```

Comment:
Jakarta Persistence and Jakarta NoSQL define entity models that you may use in Jakarta Data.

Reply:
Good catch. I missed that one. It is corrected in https://github.com/jakartaee/data/pull/114/commits/c92a8af19dc09d4514081b2cb655491361d9a466